service/configinit.go

```diff
@@ -0,0 +1,530 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service // import "go.opentelemetry.io/collector/service"
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
```

Comment:
Any reasons to use stdlib here instead of zap.Logger?

Reply:
I think since we only want to show error and warning logs, stdlib should be okay?

Reply:
Consistency is better here. Unless there's a strong reason to use stdlib, use the logging library we use elsewhere in the code base.

Reply:
Got it. I am now using zap.Logger